PermissionController/src/com/android/permissioncontroller/permission/utils/HealthPermissionUtils.java

```diff
@@ -0,0 +1,49 @@
+package com.android.permissioncontroller.permission.utils;
+
+import static android.health.connect.HealthPermissions.HEALTH_PERMISSION_GROUP;
+
+import android.content.Context;
+import android.content.Intent;
+import android.content.pm.PackageManager;
+import android.content.pm.ResolveInfo;
+import android.health.connect.HealthConnectManager;
+import android.os.UserHandle;
+import android.util.Log;
+
+import androidx.annotation.NonNull;
+
+import java.util.List;
+
+public class HealthPermissionUtils {
+
+    public static boolean isHealthPermissionAndShowing(@NonNull String permGroupName) {
+        return Utils.isHealthPermissionUiEnabled() && HEALTH_PERMISSION_GROUP.equals(permGroupName);
```

Comment:
`isHealthPermissionUiEnabled()` is much slower than `String.equals()` check. It's better to call `String.equals()` first to make this method faster in the common case.